app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/level/AddLevelForm.kt

```diff
@@ -28,7 +29,7 @@ import org.koin.android.ext.android.inject
 import kotlin.math.ceil
 import kotlin.math.floor
 
-class AddLevelForm : AbstractOsmQuestForm<String>() {
+class AddLevelForm(private val includeThings: Boolean) : AbstractOsmQuestForm<String>() {
```

Comment:
Unfortunately, Android Fragments must have zero constructor parameters. For other forms, we've used inheritance instead, see e.g. `AAddLocalizedNameForm`